tests/DevBetterWeb.Vimeo.Tests/Helpers/GetStatusAnimatedThumbnailServiceBuilder.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Ardalis.ApiCaller;
+using DevBetterWeb.Vimeo.Services.VideoServices;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace DevBetterWeb.Vimeo.Tests.Helpers
+{
+  public class GetStatusAnimatedThumbnailServiceBuilder
+  {
+    public static GetStatusAnimatedThumbnailService Build(HttpService httpService)
+    {
+      var logger = new Mock<ILogger<GetStatusAnimatedThumbnailService>>().Object;
+
+      return new GetStatusAnimatedThumbnailService(httpService, logger);
```

Comment:
More factory methods we shouldn't need? I'd just do this in the test constructor typically.